css/partials/_study-spaces.scss

```diff
@@ -64,4 +64,85 @@
 	margin-left: 15px;
 	font-size: 11px;
 	line-height: 14px;
-}
\ No newline at end of file
+}
+
+//
+// Begin new styles.
+//
+
+.study-space {
+	display: flex;
+	align-items: flex-start;
+	width: 100%;
+	margin-bottom: 50px;
```

Comment:
remove margin and use padding-top: 30px; to have the zebra background match. 

with margin:
![margin](https://cloud.githubusercontent.com/assets/4327102/15405403/aecd1d3c-1dcf-11e6-8dc6-d8267deffa5c.png)

with padding:
![padding](https://cloud.githubusercontent.com/assets/4327102/15405410/b49e279c-1dcf-11e6-9b62-e667be8e56d7.png)